src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -814,6 +815,11 @@ PooledHashSet<CaptureId> adjustAndGetEntryState(ArrayBuilder<PooledHashSet<Captu
                         longLivedIds.Add(id);
                     }
 
+                    if (reference.IsInitialization)
+                    {
+                        state.Add(id);
```

Comment:
>state.Add(id); [](http://example.com/codeflow?start=24&length=14)

I think we should assert that ```Add``` returns true similar to how we do that on line 413.